File: 1. Iniciante/URI1043.cpp
#include <iostream>
#include <cstdio>
#include <math.h>

using namespace std;

int main() {
    double A, B, C, perimetro, area;

    cin >> A >> B >> C;

    // não é triguanulo
    if ((A>=B+C || B>=C+A || C>=A+B)) {
        area = (((A + B) * C) / 2);
        printf("Area = %.1f\n", area);
    } else{
        perimetro = ((A) + (B) + (C));
        printf("Perimetro = %.1f\n", perimetro);
    }
    return 0;
}

File: 1. Iniciante/URI1043.py
x, y,z = input().split(" ")

A= float(x)
B = float(y)
C = float(z)

#não é triguanulo
if A>=B+C or B>=C+A or C>=A+B:
    area = (((A + B) * C) / 2)
    print("Area = %.1f" %area)
else:
    perimetro = ((A) + (B) + (C))
    print("Perimetro = %.1f" %perimetro)
